### PR TITLE
Apposite 20487 d mod07 lm

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD07_LAB_MANUAL.md [hunk 1]
```diff
@@ -1,241 +1,239 @@
...
+   ```url
+   [Repository Root]\AllFiles\Mod07\Labfiles\Lab1\Assets\france.jpg
+   ```
 
 #### Task 3: Run a script to upload multiple aircraft images
 
-1. Open **PowerShell** and run the following **uploadPhotos** script in **Assets** folder.
-2. To run the script is needed **key** from the **Storage account**.
-    >**Note:** The script is to upload images folder to the **Blob** container.
+1. Open Azure PowerShell, and then, in the **Assets** folder, run the following **uploadPhotos** script.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD07_LAB_MANUAL.md]: there is no "following" script

Comment (AviAvni) [reply Instructions/20487D_MOD07_LAB_MANUAL.md]: the script is in https://github.com/MicrosoftLearning/20487D-Developing-Microsoft-Azure-and-Web-Services/tree/master/AllFiles/Mod07/Labfiles/Lab1/Assets

Comment (AviAvni) [reply Instructions/20487D_MOD07_LAB_MANUAL.md]: @VinayApposite Any updates?

Comment (VinayApposite) [reply Instructions/20487D_MOD07_LAB_MANUAL.md]: Fixed.

**Comment by barucht** [comment]
Ready for QA

**Pull request merged**